index.js

```diff
@@ -352,9 +354,14 @@ class Raspi extends EventEmitter {
       pin: normalizedPin,
       pullResistor: pinInstance.pullResistor
     };
-    if (this[pins][normalizedPin].supportedModes.indexOf(mode) == -1) {
+    const isModeSupported = this[pins][normalizedPin].supportedModes.indexOf(mode) > -1;
+
+    if (mode === PWM_MODE && isModeSupported === false) {
```

Comment:
Given the `enableSoftwarePwm` flag, and the removal of `SOFT_PWM_MODE` this logic, and the logic further down will need to be refactored.